### PR TITLE
GDPR, including the Client ID

### PR DESCRIPTION
**Pull request by morsssss**
@bpaduch , can you help me see whether there's a build issue here?

This is just the site changes, not the ClientID stuff.  Much had changed since I did this, so I merged in the latest from master... there's still one change that's not mine that you may not want (the one toward the end of head.html)

### CODE REVIEW

**Comment by bpaduch** [comment]
@morsssss - experiencing no build errors/issues.  Make sure you're using the latest [yarn, grow, etc](https://github.com/ampproject/docs#install).  Also, when you fix the other issues, deploy to "custom" so can be reviewed by Rudi and Eric.

**Comment by morsssss** [comment]
@bpaduch, please check this out again!

Note though that this you can't dismiss the dialog box in an Incognito window.  This might be a bug report for AMP...

**Comment by morsssss** [comment]
Actually, I tried this again, and now it works in Incognito. Strange. Maybe it was an issue with the device simulator...

**Comment by bpaduch** [comment]
@pbakaus - please review.  I deployed it for Ben to the Custom site at: https://ampproject-custom.firebaseapp.com/

**Comment by bpaduch** [comment]
Screenshots on local:

Desktop
localhost_8080_

Mobile
localhost_8080_ pixel 2

File: views/partials/head.html [hunk 1]
```diff
@@ -80,7 +82,7 @@
       {% include "/assets/css/case-study.min.css" %}
     {% elif doc.view == '/views/list_page.html' or doc.url.path.endswith('/latest/') %}
       {% include "/assets/css/list.min.css" %}
-    {% elif doc.view == '/views/blog_detail.html' or doc.url.path.find('/roadmap/') != -1 %}
+    {% elif doc.view == '/views/blog_detail.html' or doc.url.path.endswith('/roadmap/') %}
```
Comment (pbakaus) [inline views/partials/head.html]: This one needs to be reverted, as the `find` syntax is needed to not break the roadmap filter pages.

Comment (morsssss) [reply views/partials/head.html]: Thanks for catching that. It's an artifact of my attempt to merge my fork to reflect changes on master :-/

**Comment by morsssss** [comment]
Ok.  @bpaduch or @pbakaus - please take a look! I reverted that unwanted change, and I also added the `<meta>` tag that's supposed to make Client ID work in GA.

**Comment by pbakaus** [comment]
LGTM, thanks!

**Pull request merged**